scripts/rmc_flags.py

```diff
@@ -139,6 +139,16 @@ def add_check_flags(make_group, add_flag, config):
     add_flag(group, "--unwinding-checks", default=True, action=BooleanOptionalAction,
              help="Turn on default unwinding checks")
 
+# Add flags for common CBMC flags
+def add_common_flags(make_group, add_flag, config):
+    group = make_group("Common flags", "Common CBMC flags handled by RMC.")
+    add_flag(group, "--object-bits", type=str,
```

Comment:
Can we add the default value for `--object-bits` so that it appears in the help message?

Reply:
The main problem with setting the variable here is that then there is no way to tell if users has set the flag themselves. It could be done but then we cannot raise an exception if the flag is set via CBMC. In that case, we would use the CBMC flag if given, but this would prevent users from changing it via the RMC flag.

If we do not want that behavior, I propose editing the help message to specify the default and adding a comment too. What do you think?

Reply:
I agree with editing the help message instead.